tests/cli/test_cli_utils.py

```diff
@@ -0,0 +1,12 @@
+from chemprop.cli.utils.parsing import parse_indices
+
+
+def test_parse_indices():
+    """
+    Testing if CLI parse_indices yields expected results.
+    """
+    splits = {"train": [0, 1, 2, 4], "val": [3, 5, 6], "test": [7, 8, 9]}
+    split_idxs = {"train": "0-2, 4", "val": "3,5-6", "test": [7, 8, 9]}
```

Comment:
```suggestion
    split_idxs = {"train": "0-2, 4", "val": "3,5-6", "test": "7-9"}
```

Reply:
If you intend to test this way, you can disregard this comment.

Reply:
Yep, I wanted to test a string with a space, a string without a space, and list.